vitals/IAlertSystem.java

```diff
@@ -0,0 +1,6 @@
+package vitals;
+
+public interface IAlertSystem {
+	public void sound_alarm(String[] message);
```

Comment:
Whenever we have a new way of alerting, we will need to change this interface. Instead, can you have sound and SMS as two implementations of IAlertSystem?

Reply:
I have updated the code according to the requirement. Thanks for the feedback sudeep.